templates/deprecated-windows.tpl.yaml

```diff
@@ -25,10 +26,6 @@ metadata:
       /objects[0].spec.template.spec.volumes
       /objects[0].spec.template.spec.networks
 
-    name.os.template.kubevirt.io/win2k19: {{ lookup('osinfo', 'win2k19').name }}
```

Comment:
I found out I was wrong, we shouldn't remove the labels, but we should change their values to `null`, otherwise when the cluster updates the templates, it leaves the labels on if they do not exist in the newer version, setting the label values to `null` would remove them.

Reply:
Done